aggregator/src/app/EthereumService.ts

```diff
@@ -302,6 +302,19 @@ export default class EthereumService {
 
       const attemptResult = await attempt();
 
+      try {
+        const result = attemptResult.value.events[0].args.results[0]; // For errors this is "Error(string)"
+        const errorArgBytesString = `0x${result.substring(10)}`; // remove methodId (4bytes after 0x)
+        const errorString = ethers.utils.defaultAbiCoder.decode(
+          ["string"],
+          errorArgBytesString,
+        )[0]; // decoded bytes is a string of the action index that errored.
+
+        console.error(errorString)
```

Comment:
I'm not sure if there's anything else we can do here other than just log out the message?  If you have thoughts let me know